src/lib/components/VideoCard/VideoCard.svelte

```diff
@@ -17,17 +34,44 @@
       console.warn("Unsupported video type; could not embed video.");
     }
   }
+
+  afterUpdate(async () => {
+    if (youtubeVideoId && (!title || !description)) {
+      const snippet = await getYoutubeVideoData(youtubeVideoId);
+      title = snippet.title;
+      description = snippet.description;
+    }
+  });
 </script>
 
-<!-- TODO: Build out card instead of just embedding video directly in page. -->
-<!-- TODO: Make embed responsive. -->
 {#if youtubeVideoId}
-  <iframe
-    width="560"
-    height="315"
-    src={`https://www.youtube.com/embed/${youtubeVideoId}`}
-    title="Embedded YouTube video"
-    frameborder="0"
-    allowfullscreen
-  />
+  {#key youtubeVideoId}
+    <div class="ldaf-video-card ldaf-video-card--{variation} {className}">
+      <div class="ldaf-video-container">
+        <iframe
+          class="ldaf-video-embed"
+          src={`https://www.youtube-nocookie.com/embed/${youtubeVideoId}`}
```

Comment:
Using the `nocookie` url is supposed to disable trackers (until the user actually watches a video).